docs/getting-started.md

```diff
@@ -0,0 +1,276 @@
+that's what we'll do here! We'll be building a simple web page which lets you enter a first and/or
+last name, and then greets you using your name. You can see a finished example [here](https://plnkr.co/edit/ahd84pIgOF7OTKgavvPP?p=preview).
+
+As you go through this guide, we encourage you to code along with it. By the end, you will have
+learned the fundamentals incorporating MDLv2 into simple sites, as well as worked with some of the
+components we have to offer.
+
+### Setting up the project
+
+First, let's set up our project. Throughout this guide, we'll assume you have a recent version of
+[NodeJS](https://nodejs.org/en/) and [npm](https://www.npmjs.com/) available on your `$PATH`. You can get the latest NodeJS download [here](https://nodejs.org/en/download/), or use a
+tool like [nvm](https://github.com/creationix/nvm) to install it.
+
+Once node is installed, create a directory for the site and install MDLv2 inside of it.
+
+```
+mkdir greeting-app
+cd greeting-app
+npm init -y # adds a package.json file into the directory
+npm install --save material-design-lite
+```
+
+This will install MDLv2 into the `node_modules` folder inside of the `greeting-app` directory.
+
+While we're at it, let's also install [live-server](http://tapiov.net/live-server/) so we can
+develop our greeting app using a local server, and have it automatically reload when we make
+changes.
+
+```
+npm install --global live-server
+```
+
+The `--global` flag tells npm to install the package globally, so that the `live-server` program
+will be available on your path.
+
+### Creating the skeleton index.html file
+
+Now that we have a sane environment set up, let's create a simple `index.html` file and include
+the assets needed for MDLv2. Put the following within `index.html` in the `greeting-app` directory:
+
+```html
+<!DOCTYPE html>
+<html class="mdl-typography">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Greeting App</title>
+    <link rel="stylesheet" href="/node_modules/material-design-lite/dist/material-design-lite.css">
+  </head>
+  <body>
+    <h1 class="mdl-typography--display1">Hello, World!</h1>
+    <button type="button" class="mdl-button mdl-button--raised mdl-button--primary">
+      Press Me
+    </button>
+  </body>
+</html>
+```
+
+To view the page, run `live-server` within the `greeting-app` directory. This will open up your
+browser to the URL which is serving our `index.html` file. You can leave `live-server` running for
+the duration of this guide.
+
+Let's take a look at a few aspects of the above HTML.
+
+* **No javascript necessary (yet)** - Because we aren't using any dynamic components, we only need
+  to include the MDLv2 CSS so that we can apply the proper css classes to our elements. With MDLv2,
+  javascript is only necessary for dynamic components whose UI needs to be made aware of events
```

Comment:
Nit: You use "Javascript" in some places and "javascript" in others. I'm happy with either, but it's probably best to be consistent.